src/libraries/Microsoft.PowerFx.Core/Public/Values/StringValue.cs

```diff
@@ -23,5 +23,10 @@ internal StringValue ToLower()
         {
             return new StringValue(IRContext.NotInSource(FormulaType.String), Value.ToLowerInvariant());
         }
+
+        public override string ToExpression()
+        {
+            return $"\"{Value}\"";
```

Comment:
>$"\"{Value}\""; [](http://example.com/codeflow?start=19&length=15)

What happens if the string contains quotes, spaces, non-printable characters like \t or CR, LF?

Reply:
yes, use one of the existing parser helpers to escape this.

Reply:
Actually ... this may get very tricky... 
Quotes can be escaped as double quotes.
Newlines can exist as-is!
But other chars would need to be Char() function. 
Like:
"abc\tdef" // C# 
"abc" & Char(9) & "def" // Power Fx